src/zjs_webusb.json

```diff
@@ -0,0 +1,27 @@
+{
+    "module": "webusb",
+    "description": "Module to advertise as a WebUSB device (only A101 has a USB driver in Zephyr currently)",
+    "require": "webusb",
+    "targets": ["arduino_101"],
+    "zephyr_conf": {
+        "all": [
```

Comment:
why is this "all"? the only target is arduino_101, there's no support for k64f, shouldn't these also be a101-specific configs?

Reply:
The DW one is truly A101-specfic, in that it uses the DW USB hardware. But the others are all software dependencies and would be needed for other platforms; so this is just forward looking to when there is more than one platform with USB support.